src/test/java/io/pravega/connectors/flink/source/reader/FlinkPravegaSplitReaderTest.java

```diff
@@ -72,43 +75,21 @@ public void testHandleSplitChangesAndFetch() throws Exception {
         reader.close();
     }
 
-    @Test
-    public void testWakeUp() throws Exception {
+    @Test(expected = Exception.class)
```

Comment:
1. This exception is too general, can we make it a more concrete exception here?
2. This method is using `Whitebox` to hack into a private variable in a class. I can understand it is easy to write, but it is a bit hacky and should be the last resort if we cannot find an alternative. The more elegant way is to create a new `TestPravegaSplitReader` class extends that class and do some cuystomization for testing purposes. Similar techniques can be seen in `FlinkPravegaReaderTest::TestableFlinkPravegaReader`